lib/user-interface/react/src/components/chatbot/components/Message.tsx

```diff
@@ -220,7 +220,7 @@ export const Message = React.memo(({ message, isRunning, showMetadata, isStreami
         if (Array.isArray(content)) {
             return content.map((item, index) => {
                 if (item.type === 'text') {
-                    return item.text.startsWith('File context:') ? <></> : <div key={index}>{getDisplayableMessage(item.text, message.type === MessageTypes.AI ? ragCitations : undefined)}</div>;
+                    return item.text.startsWith('File context:') ? <div key={index}></div> : <div key={index}>{getDisplayableMessage(item.text, message.type === MessageTypes.AI ? ragCitations : undefined)}</div>;
```

Comment:
Looks like we're repeating ourselves, so maybe just something like below?

```suggestion
                    return <div key={index}>{ item.text.startsWith('File context:') && getDisplayableMessage(item.text, message.type === MessageTypes.AI ? ragCitations : undefined)}</div>;
```